pkg/platform/aws/aws.go

```diff
@@ -88,6 +88,11 @@ type config struct {
 	KubeAPIServerExtraFlags  []string
 }
 
+const (
+	// Name represents AWS platform name as it should be referenced in function calls and configuration.
+	Name = "aws"
+)
+
 // init registers aws as a platform
 func init() {
 	platform.Register("aws", NewConfig())
```

Comment:
Can you please use the constant here also? Also, why not merge this commit and aks one?

```suggestion
platform.Register(Name, NewConfig())
```

Reply:
Now that I think of it, as you removed init function in the next commit. I would say it is un-necessary. But I would suggest merging this commit with AKS one.

Reply:
Yes, I should've done this. Will fix regardless.